src/main/java/reposense/parser/AuthorConfigCsvParser.java

```diff
@@ -73,14 +73,14 @@ protected void processLine(List<RepoConfiguration> results, String[] elements)
 
 
     /**
-     * Gets an existing {@code RepoConfiguration} from {@code results} if {@code location} and {@code branch} matches.
-     * Otherwise adds a newly created {@code RepoConfigurtion} into {@code results} and returns it.
+     * Gets an existing {@code AuthorConfiguration} from {@code results} if {@code location} and {@code branch} matches.
+     * Otherwise adds a newly created {@code AuthorConfiguration} into {@code results} and returns it.
      *
      * @throws InvalidLocationException if {@code location} is invalid.
      */
-    private static RepoConfiguration getRepoConfiguration(
-            List<RepoConfiguration> results, String location, String branch) throws InvalidLocationException {
-        RepoConfiguration config = new RepoConfiguration(new RepoLocation(location), branch);
+    private static AuthorConfiguration getAuthorConfiguration(
+            List<AuthorConfiguration> results, String location, String branch) throws InvalidLocationException {
+        AuthorConfiguration config = new AuthorConfiguration(new RepoLocation(location), branch);
         int index = results.indexOf(config);
```

Comment:
And here, instead of using `indexOf` to find an 'equal' `AuthorConfiguration`, we can just iterate through each `AuthorConfiguration` and check if the location and branch is the same, and return that config.